docs/test-plans/003-PayForData-GetSharesByNamespace/test-cases/tc-001-da-nodes-pfd.md

```diff
@@ -0,0 +1,41 @@
+# Test-Case #001 - DA Nodes can submit pfds
+
+## Pre-Requisites:
+
+1. Every validator has enough funds in the account
+2. The chain has created the first block
+3. Every validator has enough peers
+4. Validators’ set is not changing during test execution
+   1. All validators are created during genesis
+5. DA Nodes' accounts are funded
+
+## Steps for each of the validators:
+
+1. Setups network with:
+   1. `I` mb of bandwidth
+   2. `J` milliseconds of network latency
+2. Generates and broadcasts
+   1. IP and Genesis Hash for DA Bridge nodes
+3. Checks the tx amount is the same as DA Nodes amount
+
+## Steps for each of the DA nodes:
+
+1. Setups network with:
+   1. `I` mb of bandwidth
+   2. `J` milliseconds of network latency
+2. Bridge nodes connect to respective Validators
+3. Full / Light nodes connect to Bridge Nodes
+4. Starts syncing the chain
+5. Generates PFD wtih
+   1. `X` kb of random data
+   2. `Y` amount of namespace id
```

Comment:
Does this amount describe the number of different NIDs?

Reply:
Yes it does